src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute is pointing to an SCSS file (`index.scss`). SCSS files need to be compiled into CSS before they can be linked in an HTML document. Make sure that you have a build process in place to compile SCSS to CSS.